pom.xml

```diff
@@ -220,6 +220,11 @@
       <artifactId>lucene-test-framework</artifactId>
       <version>${LUCENE_VERSION}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.lucene</groupId>
+      <artifactId>lucene-analyzers-smartcn</artifactId>
+      <version>6.3.0</version>
```

Comment:
change to `${LUCENE_VERSION}`?

Reply:
I did that but I got a wired error during CI (`${LUCENE_VERSION} is not a version number...`). I can compile that in my local machine. Let me try again.